src/System.Text.JsonLab/System/Text/Json/JsonWriterUtf8.cs

```diff
@@ -11,9 +11,12 @@ namespace System.Text.JsonLab
 {
     public struct JsonWriterUtf8
```

Comment:
I am leaving this as JsonWriterUtf8 for now to make reviewing the merge easy. Once this PR is done, I will rename the type back to JsonWriter.